src/components/TodoInfo/TodoInfo.spec.tsx

```diff
@@ -1,115 +1,115 @@
 import { mount } from '@cypress/react18';
-import { TodoInfo } from './TodoInfo';
+import { TodoInfo } from "./TodoInfo";
 
-describe('TodoInfo', () => {
-  it('should have a correct title', () => {
+describe("TodoInfo", () => {
+  it("should have a correct title", () => {
     const user1 = {
       id: 1,
-      name: 'Leanne Graham',
-      username: 'Bret',
-      email: 'Sincere@april.biz',
+      name: "Leanne Graham",
+      username: "Bret",
+      email: "Sincere@april.biz",
     };
 
     const todo = {
       userId: 1,
       id: 1,
-      title: 'Learn HTML',
+      title: "Learn HTML",
       completed: true,
       user: user1,
     };
 
     mount(<TodoInfo todo={todo} />);
 
-    cy.get('.TodoInfo__title').should('have.text', 'Learn HTML');
+    cy.get(".TodoInfo__title").should("have.text", "Learn HTML");
   });
 
-  it('should add TodoInfo--completed class for a completed todo', () => {
+  it("should add TodoInfo--completed class for a completed todo", () => {
     const user1 = {
       id: 1,
-      name: 'Leanne Graham',
-      username: 'Bret',
-      email: 'Sincere@april.biz',
+      name: "Leanne Graham",
+      username: "Bret",
+      email: "Sincere@april.biz",
     };
 
     const completedTodo = {
       userId: 1,
       id: 1,
-      title: 'Learn HTML',
+      title: "Learn HTML",
       completed: true,
       user: user1,
     };
 
     mount(<TodoInfo todo={completedTodo} />);
 
-    cy.get('.TodoInfo').should('have.class', 'TodoInfo--completed');
+    cy.get(".TodoInfo").should("have.class", "TodoInfo--completed");
```

Comment:
Ensure that the `TodoInfo` component adds the `TodoInfo--completed` class correctly when a todo is completed. Verify that the component implementation aligns with this test expectation.